routing.rst

```diff
@@ -379,7 +379,28 @@ You can also use these functions:
 ``service(string $alias)``
     Returns a routing condition service.
     You'll have to add the ``#[AsRoutingConditionService]`` attribute or ``routing.condition_service``
-    tag to your service if you want to use it in the condition.
+    tag to your service if you want to use it in the condition. Example:
+    
+    .. code-block:: php
+    
+        // Controller:
```

Comment:
```suggestion
        // using an alias:
```

Reply:
I reworded to "Controller (using an alias)", cause the important part is IMO *where* to put this (=> Controller)